src/audio/Kconfig

```diff
@@ -134,6 +134,14 @@ config COMP_CROSSOVER
 	  split a signal into two or more frequency ranges, so that the outputs
 	  can be sent to drivers that are designed for those ranges.
 
+config COMP_DRC
+	bool "Dynamic Range Compressor component"
+	default n
+	help
+	  Select for Dynamic Range Compressor (DRC) component. A DRC can be used
+	  to reduce the volume of loud sounds thus compressing an audio signal's
+	  dynamic range.
```

Comment:
I think it can also amplify silent sounds (in addition to limit output) based on the defined compression curve. Right?